### ISSUE TITLE
Error formatter used with apollo omits the error message

### ISSUE THREAD

**Issue by focusaurus** [reactioncommerce/reaction#5060]
## Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description

When formatting and logging errors in [util/getErrorFormatter](https://github.com/reactioncommerce/reaction/blob/6c8e8066e7cf36057f93bbf4f08e836676c44896/imports/node-app/core/util/getErrorFormatter.js#L29), we log the object without the error message. Later we set the error message, but when we actually log this object, there's nothing useful:

```
{
  "errorId": "cjtiy93co09vj011h9mzw5y8n",
  "hostname": "reaction-core-74678fbc6-pndht",
  "level": 50,
  "msg": "",
  "name": "Reaction",
  "pid": 1,
  "time": "2019-03-21T18:08:47.928Z",
  "v": 0
}
```

## Steps to Reproduce

Set `REACTION_LOG_FORMAT` to `json` and watch the logs. (Unsure of root case to reproduce, because of missing error message, but I believe it's graphql/apollo related). We are currently seeing this every 1s or so in a particular deployment.

## Possible Solution

I think if we just move the line that logs this lower to after where the `message` is populated, that should do the trick. We might want to pass `err.message` explicitly as the 2nd bunyan argument and with a greppable fallback too.

**Comment by janus-reith** [comment]
Probably related: https://github.com/reactioncommerce/reaction/issues/5006 ?

**Comment by aldeed** [comment]
@focusaurus I agree there's something not quite right with the gql error logging. Hard to debug where errors originate sometimes.

It makes sense to move the `Logger.error` lower. Apollo does do some of its own voodoo on the error before our formatter function is even hit, so hopefully it's not an issue on their side. I believe `err` is a GraphQL-style error object and `originalError` is the original `Error` or `Error` subclass that was thrown deeper in the code. The way `GraphQL` works is that multiple errors might be hit from multiple resolvers, and they are all combined into an `errors` array, and each error in that array goes through the formatter before being added to the response that the client sees.

If you can do something that fixes #5006 at the same time, so much the better. (Might be exact same issue.)

**Comment by focusaurus** [comment]
OK going to mark this as a dupe and continue discussing in #5006.
